server/cluster/cluster.go

```diff
@@ -295,11 +297,74 @@ func (c *RaftCluster) Start(s Server) error {
 	go c.runMinResolvedTSJob()
 	go c.runSyncConfig()
 	go c.runUpdateStoreStats()
+	go c.startGCTuner()
 
 	c.running.Store(true)
 	return nil
 }
 
+// startGCTuner
+func (c *RaftCluster) startGCTuner() {
+	defer logutil.LogPanic()
+	defer c.wg.Done()
+
+	tick := time.NewTicker(10 * time.Second)
+	defer tick.Stop()
+	totalMem, err := memory.MemTotal()
+	if err != nil {
+		log.Fatal("fail to get MemTotal:%s", zap.Error(err))
+	}
+	log.Info("MemTotal", zap.Uint64("totalMem", totalMem))
+	cfg := c.opt.GetPDServerConfig()
+	enableGCTuner := cfg.EnableGOGCTuner
+	memoryLimitBytes := uint64(float64(totalMem) * cfg.ServerMemoryLimit)
+	gcThresholdBytes := uint64(float64(memoryLimitBytes) * cfg.GCTunerThreshold)
```

Comment:
In this part, I mainly referred to the implementation in TiDB: https://github.com/pingcap/tidb/blob/85c48edbb63bebc25d7e56ab593293a6385a84fb/sessionctx/variable/sysvar.go#L863-L874
In this form, I think it would be very convenient for gctuner to cooperate with the global memory limiter.